pkg/oauth/api/validation/validation.go

```diff
@@ -294,3 +294,19 @@ func ValidateScopes(scopes []string, fldPath *field.Path) field.ErrorList {
 
 	return allErrs
 }
+
+func ValidateSerializedRedirectReference(sref *api.SerializedRedirectReference) field.ErrorList {
```

Comment:
Are we expecting users to actually fill out any of the fields?  Won't this require them to give us a `metadata.Name`?